Source/radiation/RadPlotvar.cpp

```diff
@@ -193,16 +200,16 @@ void Radiation::save_flux_in_plotvar(int level, const MultiFab& Snew,
             }
 
             int ifix = iflx;
-            int ifox = nflx;
+            int ifox = icomp_flux;
 
 #if AMREX_SPACEDIM >= 2
             int ifiy = iflx + NGROUPS;
-            int ifoy = nflx + NGROUPS;
+            int ifoy = icomp_flux + NGROUPS;
 #endif
 
 #if AMREX_SPACEDIM == 3
             int ifiz = iflx + NGROUPS * NGROUPS;
-            int ifoz = nflx + NGROUPS * NGROUPS;
+            int ifoz = icomp_flux + NGROUPS * NGROUPS;
```

Comment:
I think that this code was wrong originally.  We dimension the flux to be of side NGROUPS * AMREX_SPACEDIM, so the offset for 3-d should be 

```
int iflz = iflx + 2 * NGROUPS;
int ifoz = icomp_flux + 2 * NGROUPS;
```

as it stands now we have a bounds error

Reply:
the dimensioning is done in `MGFLDRadSolver.cpp` line 186

Reply:
ok, fixed